packages/clay-empty-state/src/index.tsx

```diff
@@ -6,35 +6,35 @@
 import classNames from 'classnames';
 import React, {useMemo, useState} from 'react';
 
-import {States} from './States';
-
 interface IProps extends Omit<React.HTMLAttributes<HTMLDivElement>, 'title'> {
+	/**
+	 * Provides alternative information for an image if a user for some reason cannot view it
+	 * The default value is 'No results found'
+	 */
+	altProps?: string;
```

Comment:
I think we can use the value of `imgProps`.

Reply:
We decided to remove the altProps and continue with `{...imgProps}`